test/cql-pytest/run.py

```diff
@@ -216,7 +216,9 @@ def run_scylla_cmd(pid, dir):
         '--skip-wait-for-gossip-to-settle', '0',
         '--logger-log-level', 'compaction=warn',
         '--logger-log-level', 'migration_manager=warn',
+        # Use lower settings for some parameters to allow faster testing
         '--num-tokens', '16',
+        '--query-tombstone-page-limit', '1000',
```

Comment:
You can also lower this at runtime, just for a single test, and avoid the risk of influencing other tests.
See here: https://github.com/scylladb/scylladb/blob/7c7eb81a66939609013b33b4f5a7672471518969/test/cql-pytest/test_tombstone_limit.py#L28-L31